10-30 20:12:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:12:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:12:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:12:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:12:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:12:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:42:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.action.ExecuteCommandAction.enable(ExecuteCommandAction.java:66)
	at org.openide.util.actions.NodeAction$DelegateAction.resultChanged(NodeAction.java:615)
	at org.openide.util.actions.NodeAction$DelegateAction.<init>(NodeAction.java:560)
	at org.openide.util.actions.NodeAction.createContextAwareInstance(NodeAction.java:357)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2762)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2858)
	at org.openide.explorer.view.TreeView.createPopup(TreeView.java:1070)
	at org.openide.explorer.view.TreeView$PopupAdapter.showPopup(TreeView.java:1558)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.maybePopup(MouseUtils.java:189)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.mousePressed(MouseUtils.java:180)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:280)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.Component.processMouseEvent(Component.java:6502)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3321)
	at java.awt.Component.processEvent(Component.java:6270)
	at java.awt.Container.processEvent(Container.java:2229)
	at java.awt.Component.dispatchEventImpl(Component.java:4861)
	at java.awt.Container.dispatchEventImpl(Container.java:2287)
	at java.awt.Component.dispatchEvent(Component.java:4687)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4832)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4489)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4422)
	at java.awt.Container.dispatchEventImpl(Container.java:2273)
	at java.awt.Window.dispatchEventImpl(Window.java:2713)
	at java.awt.Component.dispatchEvent(Component.java:4687)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:707)
	at java.awt.EventQueue.access$000(EventQueue.java:101)
	at java.awt.EventQueue$3.run(EventQueue.java:666)
	at java.awt.EventQueue$3.run(EventQueue.java:664)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.awt.EventQueue$4.run(EventQueue.java:680)
	at java.awt.EventQueue$4.run(EventQueue.java:678)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:677)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:162)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:211)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
10-30 20:42:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:42:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:42:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:42:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:42:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:42:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:43:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.action.ExecuteCommandAction.enable(ExecuteCommandAction.java:66)
	at org.openide.util.actions.NodeAction$DelegateAction.resultChanged(NodeAction.java:615)
	at org.openide.util.actions.NodeAction$DelegateAction.<init>(NodeAction.java:560)
	at org.openide.util.actions.NodeAction.createContextAwareInstance(NodeAction.java:357)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2762)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2858)
	at org.openide.explorer.view.TreeView.createPopup(TreeView.java:1070)
	at org.openide.explorer.view.TreeView$PopupAdapter.showPopup(TreeView.java:1558)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.maybePopup(MouseUtils.java:189)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.mousePressed(MouseUtils.java:180)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:280)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.Component.processMouseEvent(Component.java:6502)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3321)
	at java.awt.Component.processEvent(Component.java:6270)
	at java.awt.Container.processEvent(Container.java:2229)
	at java.awt.Component.dispatchEventImpl(Component.java:4861)
	at java.awt.Container.dispatchEventImpl(Container.java:2287)
	at java.awt.Component.dispatchEvent(Component.java:4687)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4832)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4489)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4422)
	at java.awt.Container.dispatchEventImpl(Container.java:2273)
	at java.awt.Window.dispatchEventImpl(Window.java:2713)
	at java.awt.Component.dispatchEvent(Component.java:4687)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:707)
	at java.awt.EventQueue.access$000(EventQueue.java:101)
	at java.awt.EventQueue$3.run(EventQueue.java:666)
	at java.awt.EventQueue$3.run(EventQueue.java:664)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.awt.EventQueue$4.run(EventQueue.java:680)
	at java.awt.EventQueue$4.run(EventQueue.java:678)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:677)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:162)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:211)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
10-30 20:43:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:43:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:43:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:43:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:43:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:43:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.action.ExecuteCommandAction.enable(ExecuteCommandAction.java:66)
	at org.openide.util.actions.NodeAction$DelegateAction.resultChanged(NodeAction.java:615)
	at org.openide.util.actions.NodeAction$DelegateAction.<init>(NodeAction.java:560)
	at org.openide.util.actions.NodeAction.createContextAwareInstance(NodeAction.java:357)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2762)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2858)
	at org.openide.explorer.view.TreeView.createPopup(TreeView.java:1070)
	at org.openide.explorer.view.TreeView$PopupAdapter.showPopup(TreeView.java:1558)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.maybePopup(MouseUtils.java:189)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.mousePressed(MouseUtils.java:180)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:280)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.Component.processMouseEvent(Component.java:6502)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3321)
	at java.awt.Component.processEvent(Component.java:6270)
	at java.awt.Container.processEvent(Container.java:2229)
	at java.awt.Component.dispatchEventImpl(Component.java:4861)
	at java.awt.Container.dispatchEventImpl(Container.java:2287)
	at java.awt.Component.dispatchEvent(Component.java:4687)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4832)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4489)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4422)
	at java.awt.Container.dispatchEventImpl(Container.java:2273)
	at java.awt.Window.dispatchEventImpl(Window.java:2713)
	at java.awt.Component.dispatchEvent(Component.java:4687)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:707)
	at java.awt.EventQueue.access$000(EventQueue.java:101)
	at java.awt.EventQueue$3.run(EventQueue.java:666)
	at java.awt.EventQueue$3.run(EventQueue.java:664)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.awt.EventQueue$4.run(EventQueue.java:680)
	at java.awt.EventQueue$4.run(EventQueue.java:678)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:677)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:162)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:211)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
10-30 20:46:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:31 jdbc[4]: null
org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at org.netbeans.modules.db.explorer.DbDriverManager.getConnection(DbDriverManager.java:131)
	at org.netbeans.modules.db.explorer.DatabaseConnection.doConnect(DatabaseConnection.java:928)
	at org.netbeans.modules.db.explorer.DatabaseConnection.access$500(DatabaseConnection.java:121)
	at org.netbeans.modules.db.explorer.DatabaseConnection$7.run(DatabaseConnection.java:994)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 13 more
10-30 20:46:37 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:37 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:37 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:37 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:37 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:37 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:46:55 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TODOLIST" не найдена
Table "TODOLIST" not found; SQL statement:
SELECT item FROM todoList [42102-169]
10-30 20:47:24 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Столбец "ITEM" не найден
Column "ITEM" not found; SQL statement:
SELECT item FROM person [42122-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:138)
	at org.h2.command.dml.Select.prepare(Select.java:799)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1114)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:70)
	at testh2.TestH2.main(TestH2.java:32)
10-30 20:47:37 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Столбец "item" не найден
Column "item" not found [42122-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:2893)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2979)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:291)
	at testh2.TestH2.main(TestH2.java:34)
10-30 20:47:57 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Столбец "item" не найден
Column "item" not found [42122-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:2893)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2979)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:291)
	at testh2.TestH2.main(TestH2.java:34)
10-30 20:50:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.action.ExecuteCommandAction.enable(ExecuteCommandAction.java:66)
	at org.openide.util.actions.NodeAction$DelegateAction.resultChanged(NodeAction.java:615)
	at org.openide.util.actions.NodeAction$DelegateAction.<init>(NodeAction.java:560)
	at org.openide.util.actions.NodeAction.createContextAwareInstance(NodeAction.java:357)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2762)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2858)
	at org.openide.explorer.view.TreeView.createPopup(TreeView.java:1070)
	at org.openide.explorer.view.TreeView$PopupAdapter.showPopup(TreeView.java:1558)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.maybePopup(MouseUtils.java:189)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.mousePressed(MouseUtils.java:180)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:280)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.Component.processMouseEvent(Component.java:6502)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3321)
	at java.awt.Component.processEvent(Component.java:6270)
	at java.awt.Container.processEvent(Container.java:2229)
	at java.awt.Component.dispatchEventImpl(Component.java:4861)
	at java.awt.Container.dispatchEventImpl(Container.java:2287)
	at java.awt.Component.dispatchEvent(Component.java:4687)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4832)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4489)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4422)
	at java.awt.Container.dispatchEventImpl(Container.java:2273)
	at java.awt.Window.dispatchEventImpl(Window.java:2713)
	at java.awt.Component.dispatchEvent(Component.java:4687)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:707)
	at java.awt.EventQueue.access$000(EventQueue.java:101)
	at java.awt.EventQueue$3.run(EventQueue.java:666)
	at java.awt.EventQueue$3.run(EventQueue.java:664)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.awt.EventQueue$4.run(EventQueue.java:680)
	at java.awt.EventQueue$4.run(EventQueue.java:678)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:677)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:162)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:211)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
10-30 20:50:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:50:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:50:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:50:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:50:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:50:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:50:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:50:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:50:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:50:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:50:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 20:50:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 21:07:07 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 21:07:07 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 21:07:07 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 21:07:07 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:229)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:316)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 21:07:07 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:317)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
10-30 21:07:07 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1364)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:586)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:310)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:384)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:196)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:112)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:188)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1411)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1991)
